src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php

```diff
@@ -86,6 +86,9 @@ public static function dataFieldsProvider()
                         'search' => 'now',
                         'searchFields' => 'title',
                         'page' => 1,
+                    ],
+                    [],
+                    [
```

Comment:
in the test above it also has to be done. the function `Request::getRequestFormat()` in 3.0 searches in the attributes for `_format`. maybe in the testcase before the format wont be requested. but i will check this.